src/main/java/com/osiris/autoplug/client/configs/SSHConfig.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2021-2023 Osiris-Team.
+ * All rights reserved.
+ *
+ * This software is copyrighted work, licensed under the terms
+ * of the MIT-License. Consult the "LICENSE" file for details.
+ */
+
+package com.osiris.autoplug.client.configs;
+
+import java.io.IOException;
+
+import com.osiris.dyml.Yaml;
+import com.osiris.dyml.YamlSection;
+import com.osiris.dyml.exceptions.DuplicateKeyException;
+import com.osiris.dyml.exceptions.IllegalKeyException;
+import com.osiris.dyml.exceptions.IllegalListException;
+import com.osiris.dyml.exceptions.NotLoadedException;
+import com.osiris.dyml.exceptions.YamlReaderException;
+import com.osiris.dyml.exceptions.YamlWriterException;
+import com.osiris.autoplug.client.tasks.SSHManager;
+import com.osiris.jlib.logger.AL;
+
+public class SSHConfig extends MyYaml {
+
+    public static SSHManager sshManager;
+    
+    public YamlSection enabled;
+    public YamlSection port;
+    public YamlSection auth_method;
```

Comment:
Tests could be created for testing different auth methods

Reply:
added some tests to validateValues